CDH-XXX: Minor UI enhancements (#1638)
SoftwareAG/c8y-docs: content/datahub/working-with-datahub-bundle/overview-offloading.md

**Offloading status**

> The offloading status is empty if the offloading has not been executed yet. For running and completed executions the start time is shown and either a calendar icon for a scheduled execution or a user icon for a manually triggered execution. For a running execution the elapsed time is additionally shown. For a completed execution the failure/success status, the number of offloaded records and the runtime are shown as well. For running and completed executions clicking on the offloading status navigates you to the detail view for that execution in the job history.

Reviewer: ```suggestion
The offloading status is empty if the offloading has not been executed yet. For running and completed executions the start time is shown and either a calendar icon for a scheduled execution or a user icon for a manually triggered execution. For a running execution the elapsed time is additionally shown. For a completed execution the failure/success status, the number of offloaded records and the runtime are shown as well. For running and completed executions click the offloading status to navigate to the detail view for that execution in the job history.
```

> The offloading status is empty if the offloading has not been executed yet. For running and completed executions the start time is shown and either a calendar icon for a scheduled execution or a user icon for a manually triggered execution. For a running execution the elapsed time is additionally shown. For a completed execution the failure/success status, the number of offloaded records and the runtime are shown as well. For running and completed executions clicking on the offloading status navigates you to the detail view for that execution in the job history.

Reviewer 2: The offloading status is empty if the offloading has not been executed yet. For running and completed executions, the start time is shown and either a calendar icon for a scheduled execution or a user icon for a manually triggered execution. For a running execution, the elapsed time is additionally shown. For a completed execution, the failure/success status, the number of offloaded records and the runtime are shown as well. For running and completed executions, clicking on the offloading status navigates you to the detail view for that execution in the job history.

> The sort control allows for sorting by successful/failed jobs. The filter control allows for filtering by execution status.

Reviewer: ```suggestion
The sort control allows you to sort by successful/failed jobs. The filter control allows you to filter by execution status.
```
